docs/conf.py

```diff
@@ -94,62 +151,83 @@
     .. role:: raw-html(raw)
         :format: html
 
-    .. note:: This page was generated from `{{ docname }} <https://github.com/azavea/raster-vision/blob/master/docs/{{ docpath }}>`__.
-"""
-#########################
+    .. note:: This page was generated from `{{ docname }} <https://github.com/azavea/raster-vision/blob/master/docs/{{ docpath }}>`__. |ColabLink|_
 
-# -- General configuration ---------------------------------------------------
+.. |ColabLink| image:: https://colab.research.google.com/assets/colab-badge.svg
+.. _ColabLink: https://githubtocolab.com/azavea/raster-vision/blob/master/docs/{{ docpath }}
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
 
-# https://read-the-docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
-import sys
-from unittest.mock import MagicMock
+.. note::
 
+    If running on Colab:
 
-class Mock(MagicMock):
-    @classmethod
-    def __getattr__(cls, name):
-        return MagicMock()
+    1.  :ref:`Install Raster Vision <install raster vision>`:
 
+        .. code-block:: shell
 
-MOCK_MODULES = ['pyproj', 'h5py', 'osgeo']
-sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+            pip install rastervision
```

Comment:
This will only work if the notebook code is compatible with the latest release of RV.

Reply:
Since there does not seem to be a solution do this. I would suggest dropping the Colab button from this PR. What do you think?

Reply:
Yeah, makes sense.

Reply:
Done. The colab-button code is preserved here for future reference: https://github.com/AdeelH/raster-vision/blob/docs2_w_colab/docs/conf.py#L154-L175.